core/src/main/resources/hudson/security/HudsonPrivateSecurityRealm/loginLink.jelly

```diff
@@ -25,7 +25,4 @@ THE SOFTWARE.
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler">
   <st:include page="/hudson/security/SecurityRealm/loginLink.jelly" />
-  <j:if test="${it.allowsSignup()}">
-    <a href="${rootURL}/signup">${%sign up}</a>
```

Comment:
I see that you removed the sign up but I seem not be able to find a substitution

Reply:
Its on the sign-in page:

<img width="499" alt="image" src="https://github.com/user-attachments/assets/3522565d-601a-45aa-9d18-e81cb69d682c" />

Reply:
thanks for clarifying